src/Viewer/services/decoder/FileManager.js

```diff
@@ -67,6 +68,7 @@ class FileManager {
         this._fileInfo = {
             name: null,
             path: null,
+            dbID: null,
```

Comment:
I feel it might be more appropriate to name this as `sessionId`, or `dbName` as discussed above.